README.md

```diff
@@ -10,21 +11,43 @@
 ember install ember-router-scroll
 ```
 
+## Basic use
+
+You need to include the addon in your `app/router.js` file so it will be injected in all your routes.
+
+```javascript
+// app/router.js 
+
+import RouterScroll from 'ember-router-scroll';
+
+const Router = EmberRouter.extend(PageTrackerMixin, RouterScroll, {
+  ...
+});
+```
+
 ### Options
-If you need to scroll to the top of an area that generates a vertical scroll bar, you can specify the id of an element of the scrollable area. Default is `window` for using the scroll position of the whole viewport. You can pass an options object in your application's `config/environment.js` file.
+If you need to scroll to the top of an area that generates a vertical scroll bar, you can specify the id of an element 
+of the scrollable area. Default is `window` for using the scroll position of the whole viewport. You can pass an options
+object in your application's `config/environment.js` file.
 
 ```javascript
 ENV['routerScroll'] = {
   scrollElement: '#mainScrollElement'
 };
+```
 
-If you want to scroll to a target element on the page, you can specify the id or class of the element on the page.  This is particularly useful if instead of scrolling to the top of the window, you want to scroll to the top of the main content area (that does not generate a vertical scrollbar).
+If you want to scroll to a target element on the page, you can specify the id or class of the element on the page.  This 
+is particularly useful if instead of scrolling to the top of the window, you want to scroll to the top of the main 
```

Comment:
Do you know why we get this diff (new line)?

Reply:
:thinking: Content seems to be the same but I'll double check and come back to you @snewcomer

Reply:
This `diff` is due to the fact that in the original README there was not a closing block code back sticks ` ``` ` so code was mixed with normal line, I've double checked info an everything is as it was on this PR @snewcomer, please see gif that points the section which I'm referring to.

![ember-rotuer-scroll](https://user-images.githubusercontent.com/12820932/40877868-98c97d14-667f-11e8-82fc-a225d4e792a5.gif)

Reply:
cool no worries.  If you don't mind making that one change, we can get this in!

Reply:
@snewcomer PR updated, please feel free to check it again. 
Thanks

Reply:
@snewcomer I have mixed the two `Installation & Usage` section as you pointed, also I think `Why Use it?` would be better at the beginning, that is like a pattern in many libraries these days.
Please let me know if you are happy with that.
Thanks